api/src/main/java/org/apache/iceberg/catalog/Namespace.java

```diff
@@ -34,7 +34,7 @@ public static Namespace empty() {
   }
 
   public static Namespace of(String... levels) {
-    if (levels.length == 0) {
+    if (null == levels || levels.length == 0) {
```

Comment:
I think this is an error, not an empty namespace. This would happen if you called `Namespace.of(nullStringArray)` right? That should be an exceptional case.

Reply:
sgtm. I added a null check